CONTRIBUTING.md

```diff
@@ -1,113 +1,111 @@
 # Contributing
```

Comment:
I recommend sending another PR to update CONTRIBUTING

Reply:
Agreed, the main purpose of this PR is to **add comments** to the source code. I'm going to revert the changes made to CONTRIBUTION.md and create a new PR for that separately.
Cc. @IsaDSC